Readme.md

```diff
@@ -22,9 +25,9 @@
 
 To get a local copy up and running follow these simple example steps.
 for SSH:
-`git clone git@github.com:saied2035/tdd-project.git`
+`git clone git@github.com:bushmusi/my-catalog.git`
 for https:
-`git clone https://github.com/saied2035/tdd-project.git`
+`git clone https://github.com/bushmusi/my-catalog.git`
```

Comment:
- [ ] I would recommend mentioning the steps or commands to run the project locally. Since this app can only run locally you have to provide clear steps so that any recruiter or user can easily run the application without proper technical knowledge.

Ex :
```
First clone the project with the command 

$ git clone https://github.com/bushmusi/my-catalog/

then go to the right folder 

$ cd my-catalog

then run the command

$ ruby main.rb
```

Reply:
Thank you, @medaminedev66, for the tip 👍